schematic/store/synapse.py

```diff
@@ -1367,15 +1367,16 @@ def format_row_annotations(
         blacklist_chars = ["(", ")", ".", " ", "-"]
 
         for k, v in row.to_dict().items():
+            keysyn = str(k)
             if annotation_keys == "display_label":
-                keySyn = str(k).translate({ord(x): "" for x in blacklist_chars})
+                kesSyn = str(k).translate({ord(x): "" for x in blacklist_chars})
```

Comment:
is this a typo? `kesSyn`? This should be caught by unit test. If not, I am thinking that the unit test needs to be more robust